src/abi_1_0.c

```diff
@@ -305,6 +305,40 @@ struct fi_info_1_7 {
         struct fid_nic_1_7        *nic;
 };
 
+struct fi_hmem_attr_1_8 {
```

Comment:
I don't think we need to have *_1_8 if we are going to 1.8. See https://github.com/ofiwg/libfabric/pull/9440/commits/b6ccf7c00861e24b1ca34ffcc85e167757605d94

IIUC you only need to define the `_1_*` for the old ABI when you bump it. And we already have fi_info_1_7 already